content/docs/practices/naming.md

```diff
@@ -61,3 +61,26 @@ pairs represents a new time series, which can dramatically increase the amount
 of data stored. Do not use labels to store dimensions with high cardinality
 (many different label values), such as user IDs, email addresses, or other
 unbounded sets of values.
+
+
+## Base units
+
+Prometheus does not have any units hard coded. For better compability, base
+units should be used. The following lists some metrics families with their base unit.
+The list is not exhaustive.
+
+| Family | Base unit | Remark | 
+| -------| --------- | ------ |
+| Time   | seconds   |        |
+| Temperature | celsius | Celsius is the most common one encountered in practice |
+| Length | meters | |
+| Bytes  | bytes | | 
+| Bits   | bytes | |
+| Percent | ratio | Values are 0-1 |
```

Comment:
Not sure I get the A_per_B part. E.g. for disk usage it is common to say 85% which would be `disk_usage=0.85` . Here using a _percent suffix may make it easier for people to differentiate from disk_usage_bytes=123456 (absolute number).
But in both cases I fail to see `disk_usage_current_to_full = 0.85`